internal/server/cmdBreakup.go

```diff
@@ -0,0 +1,16 @@
+package server
+
+import (
+	"github.com/dicedb/dice/internal/cmd"
+	"github.com/dicedb/dice/internal/comm"
+	"github.com/dicedb/dice/internal/shard"
+)
+
+// Since PING doesn't require special logic for sending to all shards
+// It just checks health on all shards
+func breakupPING(mgr *shard.ShardManager, redisCmd *cmd.RedisCmd, c *comm.Client) (cm []cmd.RedisCmd) {
```

Comment:
It doesn't make sense for breakup that will be for Global command or Single shard commands, to send []cmd.RedisCmd of more than size 1. This should just insert a copy of redisCmd in the cm array and return.

Reply:
Thanks, missed to remove it. Removed